### ISSUE TITLE
apache license ok, I'm a licensee, who's the licenser?

### ISSUE THREAD

**Issue by ghost** [universAAL/platform#44]
I mean, who has the copyright?

look at the [platform license](https://github.com/universAAL/platform/blob/master/LICENSE):

> universAAL is distributed with the Apache License, Version 2.0, January 2004,
> as specified under https://www.apache.org/licenses/LICENSE-2.0.

Comparing with the [info here](https://en.wikipedia.org/wiki/Software_license), I can see [apache web server](https://en.wikipedia.org/wiki/Apache_HTTP_Server) is under "Permissive FOSS type", and actually with the same [apache license 2.0](https://en.wikipedia.org/wiki/Apache_License) uAAL comes with, which ["retains copyright"](https://en.wikipedia.org/wiki/Copyright).

So: who owns the copyright here?

regards,
corrado.

**Comment by Alfiva** [comment]
In every single separate code project in the repositories there is a NOTICE file describing the copyright of each of those.

**Comment by ghost** [comment]
I sampled a few, looks like it's the universities or other research centers who [formed the uAAL alliance](http://www.universaal.info/blog/post/4960/Major-milestone-achievedLegal-documents-for-registering-the-uIC-as-a-legal-entity-submitted-on-October-24th/), right?

Now, given the ECB is buying junk bonds for low yields, what about having them investing in something more meaningful?

That could help the alliance funding next versions, including support for the "frozen 3.4.1", updated docs, a "complete SDK", and so on...

PS: I'm NOT joking.


cheers
corrado

**Comment by Alfiva** [comment]
yes, at the time there was no uAAL standalone entity that could copyright these things so each project belongs to whoever developed it in the first place. Only now that we have the uAAL Coalition in place that could be possible, but these kind of debates now belong there. I will mention this discussion next time we talk.

**Comment by ghost** [comment]
well,
I guess next steps are:
 - current owners to transfer copyrights to the coalition
 - update licenses (maybe a single file per repo)
 - coalition to find investors

Given the only restriction with Permissive FOSS is in the "Right to distribute" being "Yes, under same license", which is almost like the "[virality](https://en.wikipedia.org/wiki/Viral_license)" of GPL, but "postponed to the deployment phase", then the "value for investors" is when [OEMs](https://en.wikipedia.org/wiki/Original_equipment_manufacturer) will want to buy the "right to redistribute under a more restrictive license", right?

I mean, let alone official branding, documentation & training activities, certifying and distributing the apps and so on.

Regards,
Corrado.
